### ISSUE TITLE
How to enable auto Backup?

### ISSUE THREAD

**Issue by sunsus** [gentics/mesh#653]
## Gentics Mesh Version, operating system, or hardware.
- v0.30.1

## Operating System
- Linux

## JVM
- Open JDK 1.8.0_100

## Problem
Automatic  backups are not wokring.

## Reproducer
We have the following configuriation:
automatic-backup.json 
```
{
  "enabled": true,
  "mode": "FULL_BACKUP",
  "exportOptions": "",
  "delay": "4h",
  "firstTime": "23:00:00",
  "targetDirectory": "backup",
  "targetFileName": "${DBNAME}-${DATE:yyyyMMddHHmmss}.zip",
  "compressionLevel": 9,
  "bufferSize": 1048576
} 
```
## Expected behaviour and actual behaviour
Backup is not done.

**Comment by Jotschi** [comment]
I think it you also need to enable to backup handler in your `orientdb-server-config.xml`

```
        <handler class="com.orientechnologies.orient.server.handler.OAutomaticBackup">
                        <parameters>
                                <parameter name="enabled" value="false" />
                                <parameter name="config" value="${ORIENTDB_HOME}/${ORIENTDB_CONFDIR_NAME}/automatic-backup.json" />
                        </parameters>
                </handler>
```

Switch the server setting to true. You also need to ensure that the graph server is being started. (Either by manually enabling it in the `mesh.yml` (via storage.startServer) or by turning clustering on)
```
<parameter name="enabled" value="true" />
```

I'll update the docs to include this information.

**Comment by sunsus** [comment]
@Jotschi  Thanks for your answer. I have changed this now. 
It this correct that the Backup-Endpoint (/api/v1/admin/graphdb/backup) with this setting is not working anymore?

**Comment by Jotschi** [comment]
@sunsus I think there is an open issue on the backup endpoint. The endpoint is currently not working while clustering is enabled. I'll keep this ticket open and let you know when this has been resolved.

**Comment by Jotschi** [comment]
@sunsus The issue with the backup endpoint has been fixed in 0.30.2. Let me know if you need any additional information. Closing this issue for now.
